modules/bitgo/src/v2/coins/eos.ts

```diff
@@ -223,6 +223,10 @@ export class Eos extends BaseCoin {
     return 1e4;
   }
 
+  get decimalPlaces(): number {
+    return 4;
```

Comment:
I'm not sure how this fixes the ticket? It will alway still be 4 since we `eos: Object.freeze(new Eos()).` The ticket seems to refer to using the token which is set to `8` as opposed to the baseCoin which is `4`. Am In understanding this correctly? @byhow

Reply:
since `CHEX` is a token, it is using the `get decimalPlaces()` method that is defined in `eosToken.ts` , which it is able to get the correct amount of configurated decimal places of `8`. 

However when it tries to call `baseUnitsToBigUnits` from token, since it is hardcoded to `4`, even the decimals are set correctly, the unit transform will always be `4` regardless of the config. 

By making `this.decimalPlaces` in the base class, I was expecting it to be overrode in the child token class. is there anything we can do to override it to use the token configured decimals?

Reply:
`eos:CHEX` implements `EosCoin` which is different than `eos` which implements `Eos`. `EosCoin` doesn't have a `get decimalPlaces()` which derives all the way back to `BaseCoin`.

Reply:
`eos` is also derived from `BaseCoin` but is used for the coin, while `eos:CHEX` is used for account tokens.

Reply:
quick question: how do you find out whether `eos:CHEX` extends on `EosCoin` since the eos tokens `EosToken` do extend on `Eos`

Reply:
@byhow Do we have an update for this one, or are we good to close this out?

Reply:
@mmcshinsky-bitgo Hi, let's close this out as this ticket is transferred to other teams now